src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java

```diff
@@ -97,6 +100,21 @@ public static boolean equals(AddressDto addressDto,
                 && StringUtils.equalsIgnoreCase(normalise(addressDto.getPostcode()), normalise(address.getPostalCode()));
     }
 
+    public static boolean equals(CompanyIdentification existing, CompanyIdentification updated) {
+        var nullValuesCheck = handleNulls(existing, updated);
+        if (nullValuesCheck != null) {
+            return nullValuesCheck;
+        }
+
+        var format = new StringJoiner(",");
+        var registerInformationFormat = format.add(updated.getPlaceRegistered()).add(updated.getPlaceRegisteredJurisdiction());
```

Comment:
Good to get this in and good naming of new field making it as clear as possible.